cmd/swarmctl/cluster/update.go

```diff
@@ -99,6 +99,11 @@ var (
 				if err != nil {
 					return err
 				}
+
+				if cePeriod < time.Duration(0) {
```

Comment:
This I **think** is already checked here: https://github.com/docker/swarmkit/blob/master/manager/controlapi/cluster.go#L26